### DISCUSSION TITLE
Another Toltec problem

### DISCUSSION THREAD

**Discussion by petr108** [toltec-dev/toltec#595] [Support] [answered]
I have the following problem. I was using Toltec with remux for several months without any problem. Everything worked fine, after the last update. But then I probably by mistake reset my RM2 (currently ver 2.12.3.606, which is supported by Toltec), when it was frozen in Yaft, where I tried to upgrade packages. Then I pressed the power button for a long time because it was the only way how to get out of it. But after the restart, the remux is unavailable, customized screens are also lost. When I tried to reenable Toltec I realized, there is no toltecctl. So I tried to install it again, but it is not possible. Could some Toltec guru advise me, on how to solve it? How I can manually uninstall Toltec, or remove its residua so that it will be possible to install it again? I am able to reproduce any necessary steps, or run scripts if provided.
![toltec](https://user-images.githubusercontent.com/107629309/174659181-648a62d6-fcdd-4452-9a68-4edb35bc4989.PNG)

**Comment by Eeems** [comment, marked as answer]
Bootstrap should be installing `toltecctl` as part of the initial work, which is what I'm seeing there. Does `/home/root/.local/bin/opkg` exist?

**Reply by petr108** [reply]
/home/root/.local/bin is empty. No opkg folder is there.
![bin](https://user-images.githubusercontent.com/107629309/174665584-7a599fed-d603-40e7-953e-b2a2134c4255.PNG)

**Reply by Eeems** [reply]
File, not folder. So that's odd. it should have retrieved it. @matteodelabre any ideas?

**Reply by petr108** [reply]
opkg works, but it is located in several another places
![opkg](https://user-images.githubusercontent.com/107629309/174667196-6ae64f3a-edc7-4d0a-b477-ae7797c5384c.PNG)

**Reply by Eeems** [reply]
Which means that it's partially installed. How did you reset your rM?

**Reply by petr108** [reply]
By holding the power button for about 15-20 seconds approximately.

**Reply by Eeems** [reply]
So you just tried to install toltec on top of itself again? Just try reinstalling the `toltec-bootstrap` package. `opkg install --force-reinstall toltec-bootstrap`. It contains `toltecctl`

**Reply by petr108** [reply]
Thank you very much for this help. Running of `opkg install --force-reinstall toltec-bootstrap` finished with errors (see below), but `toltecctl` was then available again. So I was able to uninstall toltec correctly and then reinstall it again. Now Toltec seems to work normally. Thanks again :-)

```
reMarkable: ~/ opkg install --force-reinstall toltec-bootstrap
Removing package toltec-bootstrap from root...
Installing toltec-bootstrap (0.2.2-1) to root...
Downloading https://toltec-dev.org/stable/rmall/toltec-bootstrap_0.2.2-1_rmall.
pk
Configuring toltec-bootstrap.
Collected errors:
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/toltec-bootstrap
list: No such file or directory.
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/xochitl.list: No
such file or directory.
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/rm2fb-client.lis
: No such file or directory.
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/display.list: No
such file or directory.
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/rm2-suspend-fix.
ist: No such file or directory.
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/toltec-bootstrap
list: No such file or directory.
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/xochitl.list: No
such file or directory.
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/rm2fb-client.lis
: No such file or directory.
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/display.list: No
such file or directory.
 * pkg_get_installed_files: Failed to open //opt/lib/opkg/info/rm2-suspend-fix.
ist: No such file or directory.
```
